Source/Core/Common/MsgHandler.cpp

```diff
@@ -91,10 +91,10 @@ bool MsgAlert(bool yes_no, MsgType style, const char* format, ...)
 
   if (!info_caption.length())
```

Comment:
While we're at it with the cleanup, thoughts on using `empty()` for a check that reads nicer (expressing intent better than the length check)?

```suggestion
  if (info_caption.empty())
```
(also: no idea why GitHub thinks this was three spaces before, when it should only be two. At least in the comment preview it shows the first space being deleted. You know what I meant for that change.)

Reply:
Done, good catch.